### DISCUSSION TITLE
Update interval param value

### DISCUSSION THREAD

**Discussion by fgsalvador** [dylandoamaral/uptime-card#81] [Q&A] [answered]
Hi

Great component, thanks for sharing!

I use It within a single panel with about 40 uptime-card instances. The view loads fine, but then It hangs...

I'm considering I could use update-interval to avoid intensive refreshing, to check if that solves my issue. However, looks this param documentation lacks of enough detail. What's its default value? How It is being meassured (unit...seconds??)

Thanks in advance!

**Comment by dylandoamaral** [comment, marked as answer]
`update_interval` is in second but it won't help you since it will just add more updates. However i think this field is useless and that it should behave like you expected it to do with a default value of 30 seconds for example. Glad that you use more than 40 uptime card bar Oo. I only use two of them to monitor my website and my influxDB server uptime xD. Sorry for my late reply I can't receive notifications from discussions. This is for sure an issue, next time please raise an issue ;). I will take that ticket as fast as possible.

**Reply by dylandoamaral** [reply]
Fix with https://github.com/dylandoamaral/uptime-card/pull/82

**Reply by dylandoamaral** [reply]
@fgsalvador  Can you tell me if it fixes your problem? I know the card is still not well optimized since my knowledge in LitElement lifecycle is near to 0 ^^'.

**Reply by fgsalvador** [reply]
Hi Dylan. Just set update_interval to 300 (5') to avoid agressive refresh and now lovelace shows much more stable. I show last two days availability for my wifi connected devs (esp32s, shellies...), so I dont need this to be real time 🙂 Thanks a lot for your time and help 🙌
